e2e/cypress/integration/integrations/builtin_commands/common_commands_spec.js

```diff
@@ -82,6 +82,57 @@ describe('I18456 Built-in slash commands: common', () => {
         });
     });
 
+    it('MM-T664 /groupmsg initial tests', () => {
+        const usernames = Cypress._.map(userGroup, 'username').slice(0, 4);
+        const usernamesFormat = [
+
+            // # Format for sending a group message:
+            // /groupmsg @[username1],@[username2],@[username3] [message]
+            `@${usernames[0]}, @${usernames[1]}, @${usernames[2]}, @${usernames[3]}`,
+
+            // # Use /groupmsg command and use a mix of @ symbols in front of some names but not all
+            // # Format notes:
+            // # Usernames do not have to contain the '@' character
+            // # Accepts spaces after or before the commas when listing usernames
+            `${usernames[0]}, @${usernames[1]} , ${usernames[2]} , @${usernames[3]}`,
```

Comment:
I'm trying to capture the format notes and the fact that we can include a message. The code is not in a shape I like. Please give suggest to make the test clearer.

Reply:
I'm good as is.